config/operator/crd/bases/scylla.scylladb.com_scyllaclusters.yaml

```diff
@@ -1611,6 +1611,9 @@ spec:
                       type: string
                     description: Pool of addresses which should be replaced by new nodes.
                     type: object
+                  sidecarVersion:
```

Comment:
>I don't thinks this is how we should handle it. Managing actions through particular field updates won't scale, neither we'll reconcile our operand correctly. At some point we should just calculate what our resources should look like (StatefulSet, services, ...), hash the inputs and apply those resources as a whole (individually by waiting for the rollouts). Then next time just compare the sha placed on existing resources to our current inputs, if they differ, apply.

That would require a major refactor, which I don't want to include in 1.1. That's why I followed current approach.

> It's a lot of code to handle updating one field in desired podSpec.

Most of the code comes from synchronizing update of multiple racks. 

API change is removed, it will be easier to remove it once we have better solution for applying upgrades.